RustFxA/FxAWebViewModel.swift

```diff
@@ -253,6 +253,10 @@ extension FxAWebViewModel {
             NotificationManager().requestAuthorization { granted, error in
```

Comment:
Since this is the only other use case for this method I would suggest also using the new async function here within a Task block so that you can remove the method that uses completion blocks.